lib/collectors/global_prometheus_collector.rb

```diff
@@ -0,0 +1,28 @@
+require "prometheus_exporter"
+require "prometheus_exporter/server"
+
+module Collectors
+  class GlobalPrometheusCollector < PrometheusExporter::Server::TypeCollector
+    SECONDS_PER_DAY = 86_400
+
+    def type
+      "locations_api_global"
+    end
+
+    def metrics
+      oldest_os_places_postcode = PrometheusExporter::Metric::Gauge.new("locations_api_oldest_os_places_postcode_age_days", "Days since last update of oldest postcode OS Places postcode")
```

Comment:
It is simpler, but probably quite a bit less meaningful. Days since last update of oldest postcode will tell us if there are problems with individual postcodes getting missed somehow, whereas the most recent time only really gives us problems with OS Places API as an entity. It's a faster metric, definitely, and easier to understand, but I don't think those outweigh the benefits of the slower metric's focus on overall freshness of data (and since our aim here anyway is to have a slower metric and less sensitive alert system, perhaps not in the spirit of the ticket).

Reply:
That makes sense, thanks for the explanation. Tracking the oldest does give you an extra catch of if a bug creeps in where some postcodes are not being updated 👍 Could we can reword the description to something a bit clearer, maybe "Days since oldest postcode was last updated via OS Places API" (could also add in the expected value e.g ~7 days).

Also notice that the current graph states 12 days, could this be a result of the bug introduced, or does this need looking into?

Reply:
The current graph on integration might be down to integration having a slower refresh rate, but yeah, I noticed that. I think once we merge this in and have it running in prod for a day or two we should take a look if it's not around 7 days.

Reply:
Message updated: "Days since oldest postcode was last updated via OS Places API (Expected in prod: ~7)"

Reply:
Awesome, thanks!